app/models/spawning_success.rb

```diff
@@ -31,6 +31,12 @@ class SpawningSuccess < ApplicationRecord
 
   validates :shl_case_number, presence: true
 
+  def self.create_from_csv_data(attrs)
+    attrs['nbr_of_eggs_spawned'] = attrs.delete('number_of_eggs_spawned_if_female')
+    attrs['shl_case_number']     = attrs.delete('shl_number')
+    new(attrs)
```

Comment:
Part of the move of the CSV-to-model conversion from the jobs to the models.

Note that if you expand the collapsed content of the class above this diff, you'll see the HEADERS mapping between CSV column names and database column names. Colocating the mapping and the methods that execute on that mapping should make maintaining them clearer.

Reply:
I think this is one of the main reasons why before this refactor we were getting confused. This makes way more sense!